server/models/user.js

```diff
@@ -90,6 +90,13 @@ module.exports = (sequelize, DataTypes) => {
       onDelete: 'CASCADE',
     });
     User.hasMany(models.CommentLike);
+
+    User.hasMany(models.Bookmark, {
+      foreignKey: 'userId',
+      otherKey: 'articleId',
+      onDelete: 'CASCADE',
+      // as: 'userBookmarks'
```

Comment:
Would you still need `as: 'userBookmarks'`? If not, I think commented out codes should be excluded